test/testfolderwatcher.cpp

```diff
@@ -92,17 +96,16 @@ class TestFolderWatcher : public QObject
 
     bool waitForPathChanged(const QString &path)
     {
-        QElapsedTimer t;
-        t.start();
-        while (t.elapsed() < 5000) {
+        Utility::ChronoElapsedTimer t;
+        while (t.duration() < 15s) {
```

Comment:
Do we need to hardcode this value here? Can we need to make it depend on the values from the other file (e.g., using a constant in the header)? Otherwise, I'd at least suggest a comment that explains how to deduce this value from the values set in the other file.